changelogs/fragments/78678-add-a-g-install-offline.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+- >-
+  ``ansible-galaxy collection install`` - add an ``--offline`` option to
+  prevent querying distribution servers (https://github.com//issues/77443).
```

```suggestion
  prevent querying distribution servers (https://github.com/ansible/ansible/issues/77443).
```
_(looks like I wasn't very attentive earlier..?)_